src/components/ClientQuote/ClientQuote.module.scss

```diff
@@ -0,0 +1,46 @@
+.main {
+  display: flex;
+  flex-direction: row;
+
+  @media (max-width: 991px) {
+    flex-direction: column-reverse;
+  }
+}
+
+.imageWrapper {
+  position: relative;
+
+  @media (max-width: 991px) {
+    overflow: hidden;
+    max-height: 50vh;
+    height: 50vh;
```

Comment:
you shouldn't need the height/width and the max values. your two options would be

```
max-height: 50vh;
height: 100%;
```

or 

`height: 50vh;`